.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Checklist
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**PR Buddy:** _@ mention a team member_
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide historical context and customer-specific info._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any AMD hardware gotchas._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
+
+**Test Cases:**
+- [ ] _List here_
+
+## Approval Checklist
+- [ ] PR Buddy: Test functionality
+- [ ] PR Buddy: Test performance
+- [ ] PR Buddy: Verify accurate documentation by testing listed steps
+- [ ] Ensure checklist criteria are met
+- [ ] Ensure CI passes
+- [ ] Verify test coverage has been added
+- [ ] Verify the CHANGELOG has been updated with a summary of the feature
```

Comment:
It doesn't seem right for every PR to update CHANGELOG, which should be a summary for each release.

Reply:
Anything that impacts a customer or another team should be logged. I'll reword to leave this up to the discretion of the maintainer.